code/__HELPERS/logging.dm

```diff
@@ -36,6 +36,22 @@
 	GLOB.STUI.admin.Add("\[[time_stamp()]]ADMIN: [text]")
 	GLOB.STUI.processing |= STUI_LOG_ADMIN
 
+/proc/check_slur(var/text, var/mob/source, var/method = "Speech")
```

Comment:
imo it'd be better to do it like check_slur(var/mob/source, var/text, var/method) as it's more natural to code in that order